lib/gymmieInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+export interface InputsProps {
```

Comment:
Why are all these props optional?

Also, this interface share a lot of props with `SportClassInterface`.
You can better create a main interface and extend both of these, e.g:

```javascript
interface GymWhateverInterface {
  id: string
  name: string
  freeSpots: number
  available: number
  year: number
  week: number
  day: number
  startTime: number
  teacher: string
}

export interface SportClassInterface extends GymWhateverInterface {
 duration: number
}

export interface InputsProps extends GymWhateverInterface {
  users?: string[]
  email?: string
  password?: string
}

```